src/Commands/SiteInstallCommand.php

```diff
@@ -126,16 +143,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->acquiaCmsCli->printLogo();
         $this->acquiaCmsCli->printHeadline();
       }
-      $site_uri = $input->getOption('uri');
+      $siteUri = $input->getOption('uri');
       // Get starterkit name from build file.
-      [$starterkit_machine_name, $starterkit_name] = $this->installTask->getStarterKitName($site_uri);
-      $helper = $this->getHelper('question');
-      if ($helper instanceof QuestionHelper) {
-        $args['keys'] = $this->askQuestions->askKeysQuestions($input, $output, $starterkit_machine_name, 'install', $helper);
+      [$starterkitMachineName, $starterkitName] = $this->installTask->getStarterKitName($siteUri);
+      // Get user input options for install process.
+      $getInstallArgs = $this->getInputOptions($input->getOptions(), 'install');
+      if (empty($getInstallArgs)) {
+        $helper = $this->getHelper('question');
+        if ($helper instanceof QuestionHelper) {
+          $args['keys'] = $this->askQuestions->askKeysQuestions($input, $output, $starterkitMachineName, 'install', $helper);
+        }
+      }
+      else {
+        $args['keys'] = $getInstallArgs;
```

Comment:
Starterkit not asking the questions.
```./vendor/bin/acms acms:install acquia_cms_community --demo-content=yes```
above command should ask question related to the DAM, GDPR and GMAP key.
```suggestion
        $helper = $this->getHelper('question');
        if ($helper instanceof QuestionHelper) {
          $args['keys'] = $this->askQuestions->askKeysQuestions($getInstallArgs, $input, $output, $name, 'build', $helper);
        }
```

Reply:
updated to ask questions